static/scss/cms/product.scss

```diff
@@ -0,0 +1,28 @@
+.faculty-block {
+  color: #505050;
+
+  .head {
+    padding: 0 10px;
+
+    .subheading {
+      text-decoration: overline;
```

Comment:
This isn't working right. The dividing line should be only one line and full width. I'm not sure if you can do that as an "underline" style on the .head, or if you need to do an horizontal rule.